### PR TITLE
Ragtag: remove optional element on input parameter

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

The AGP files are required when running _Ragtag merge_.

### CODE REVIEW

File: tools/ragtag/ragtag.xml [hunk 1]
```diff
@@ -236,7 +236,7 @@
             </when>
             <when value="merge">
                 <param name="assembly_fasta" type="data" format="fasta" label="Assembly FASTA file"/>
-                <param name="scaffold_files" type="data" format="agp" multiple="true" optional="true" label="Scaffold AGP files"/>
+                <param name="scaffold_files" type="data" format="agp" multiple="true" label="Scaffold AGP files" help="At least two AGP files are required."/>
```
Comment (bgruening) [inline tools/ragtag/ragtag.xml:239]: https://docs.galaxyproject.org/en/latest/dev/schema.html#id43

can you try to add min=2 here?

Comment (gallardoalba) [reply tools/ragtag/ragtag.xml:239]: Ah sure, I didn't know it was possible. Thanks!

File: tools/ragtag/ragtag.xml [hunk 2]
```diff
@@ -272,70 +272,70 @@
     </inputs>
     <outputs>
         <!--Correct mode outputs-->
-        <data format="paf" name="correct_paf" from_work_dir="ragtag.correct.asm.paf" label="${tool.name} on ${on_string}: PAF">
```
Comment (bernt-matthias) [inline tools/ragtag/ragtag.xml:275]: Would prefer to keep `{tool.name} on ${on_string}` together and add the additional string after the colon.

Comment (bernt-matthias) [reply tools/ragtag/ragtag.xml:275]: But this is my personal preference in favor of consistency.

**Review by bgruening — APPROVED** [review]

**Pull request merged**